docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -86,27 +86,20 @@ It also enables you to **stack** them on top of one another (although this is di
 
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-1.1/#alert)
 
-- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `alert` through the SnackbarContent `role` prop. This is the default role.
-- If a Snackbar requires focus to close it, then content authors should use the `role` of `alertdialog`.
+- By default, the toast won't auto-hide. However, if you decide to use the `autoHideDuration` prop, it's recommended giving users [enough time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html).
+- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `alert="role"` through the SnackbarContent. This is the default role.
```

Comment:
```suggestion
- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `role="alert"` through the SnackbarContent. This is the default role.
```

It seems like the role and auto-hide behaviour should match here.